platform/android/java_godot_lib_jni.cpp

```diff
@@ -424,8 +424,7 @@ class JNISingleton : public Object {
 		Map<StringName, MethodData>::Element *E = method_map.find(p_method);
 		if (!E) {
 
-			r_error.error = Callable::CallError::CALL_ERROR_INVALID_METHOD;
-			return Variant();
+			return Object::call(p_method, p_args, p_argcount, r_error);
```

Comment:
@DrMoriarty Why change the return value here?

Reply:
I'm not sure about it. I call Object::call in order to use standard binds ob Object. If they called by another method I can change this.